mappings/net/minecraft/block/dispenser/ItemDispenserBehavior.mapping

```diff
@@ -1,11 +1,18 @@
 CLASS fu net/minecraft/block/dispenser/ItemDispenserBehavior
-	METHOD a dispenseItem (Lbhi;Lbcb;ILfa;Lfk;)V
+	METHOD a spawnItem (Lbhi;Lbcb;ILfa;Lfk;)V
```

Comment:
Is this change just to be in line with `spawnParticles` ? The method does dispense the item.

Reply:
The semantics of the methods in that class are actually rather interesting. I agree that this is likely a poor name choice I've made, but I don't think `dispenseItem` would be a good choice either, as it is too ambiguous alongside `dispense` (deobfuscated name) with identical signature. The difference is this method does not produce a sound or particles. Maybe it should be `dispenseSilently`?

Reply:
Changed to `dispenseSilently` in 176d9bc, marking as resolved